terraform/rds.tf

```diff
@@ -1,11 +1,42 @@
 resource "aws_rds_cluster" "wp-db" {
-  cluster_identifier      = "wp-db"
-  engine                  = "aurora-mysql"
-  engine_version          = "5.7.mysql_aurora.2.03.2"
-  availability_zones      = ["ap-southeast-2a", "ap-southeast-2b", "ap-southeast-2c"]
-  database_name           = "wpdb"
-  master_username         = "wpadmin"
-  master_password         = "iammradmin"
-  backup_retention_period = 5
-  preferred_backup_window = "07:00-09:00"
+  
+    cluster_identifier            = "wp-db"
+    database_name                 = "wpdb"
+    availability_zones            = ["ap-southeast-2a", "ap-southeast-2b"]
```

Comment:
this should be a variable to be used both here and n the VPC

Reply:
Hey Kiko,
I initally wanted to use 
` data "aws_availability_zones" "iac-azs" {
    state = "available"
}`
From the VPC.
But when trying, I recieved an error that the RDS Cluster didn't support **ap-southeast-2c**
So I manually entered these two.
I can create a list variable with 2a / 2b inside but it won't be the same as the VPC

Reply:
`data "aws_availability_zones" "iac-azs" { state = "available" }` is not the best way to define where your resources will run, because depending on what you're doing, a resource running on a specific AZ will not be able to be used by another resource running on other AZ and by doing this way, you don;t have control on where things are running.

Ideally you should have a variable(which you already have) that will contain the AZs that you want to deploy your resources. You can have a default value for this variable, so you don't need to add it to every resource you create.